### ISSUE TITLE
var {name, time, args} = obj; gives error  Unexpected token ,

### ISSUE THREAD

**Issue by phzbox** [eslint/eslint#2703]
`var {name, time, args} = obj;` returns `error  Unexpected token ,`

I have `"destructuring", true,` in my .eslintrc

**Comment by hzoo** [comment]
Maybe you didn't put it under `ecmaFeatures`?

```js

"ecmaFeatures": {
    "destructuring": true
}
```

**Comment by ilyavolodin** [comment]
Can you provide more details? What is console output? What version of eslint are you using? Are you using `babel-eslint`? Copy of you .eslintrc file.

**Comment by phzbox** [comment]
eslint version: `v0.22.1`

.eslintrc:
```
{
  "ecmaFeatures": {
    "blockBindings": true,
    "forOf": true,
    "jsx": true,
    "arrowFunctions": true,
    "destructuring", true,
    "templateStrings": true,
    "spread": true
  },
  "rules": {
    "semi": 2,
    "quotes": [1, "single", "avoid-escape"],
    "strict": false,
    "brace-style": true,
    "comma-style": true,
    "indent": [2, 2],
    "newline-after-var": [1, "always"],
    "no-underscore-dangle": false
  },
  "env": {
    "node": true
  }
}
```

Example file:

```
$> cat test.js

var obj = {a:1, b:2};

var {a,b} = obj;

console.log(a,b);
```

```
$> eslint test.js

3:7  error  Unexpected token ,

✖ 1 problem (1 error, 0 warnings)
```

I'm pretty sure the .eshintrc is being correctly parsed because this works fine:

```
var plusOne = (i) => i + 1;
console.log(plusOne(1));
```

and if I remove `arrowFunctions` then it gives an error (`error  Unexpected token >`)

**Comment by xjamundx** [comment]
In `v0.21.2` I observe no errors when I run this:

````
eslint --reset --no-eslintrc --env es6 test.js 
```

But I also *can* confirm the behavior he is saying with his `.eslintrc` file. 

Something funny is probably going on....

**Comment by xjamundx** [comment]
Oh right here `"destructuring", true,` the comma instead of the `:`

**Comment by phzbox** [comment]
Damn. It's working perfectly with the : ;-) 

I'm glad it's not a sneaky bug.

Cheers, sorry about that.
